mrtarget.yml

```diff
@@ -0,0 +1,90 @@
+#default configuration 
+
+#note: ConfigArgParse doesn't support splitting lists over multiple lines
+#therefore viewing this file with word wrap is recommended!
+
+#general configuration
+log-config: mrtarget/resources/logging.ini
+log-level: INFO
+release-tag: master
+
+#evidence validation
+schema: https://raw.githubusercontent.com/opentargets/json_schema/1.5.0/opentargets.json
+input-file: ["https://storage.googleapis.com/ot-releases/18.12/evidences/atlas-2018-11-20.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/chembl-28-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/cosmic-27-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/europepmc-29-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/eva-01-10-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/gene2phenotype-29-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/genomics_england-02-10-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/gwas-28-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/intogen-23-07-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/phenodigm-12-10-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/phewas_catalog-28-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/progeny-23-07-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/reactome-03-09-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/slapenrich-29-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/sysbio-28-11-2018.json.gz", "https://storage.googleapis.com/ot-releases/18.12/evidences/uniprot-15-11-2018.json.gz"]
+
+#reactome
+reactome-pathway-data: https://storage.googleapis.com/open-targets-data-releases/19.02/input/annotation-files/ReactomePathways-2019-01-09.txt
+reactome-pathway-relation: https://storage.googleapis.com/open-targets-data-releases/19.02/input/annotation-files/ReactomePathwaysRelation-2019-01-09.txt
+
+#uniprot
+uniprot-uri: https://storage.googleapis.com/ot-releases/18.12/annotations/uniprot-20181130.xml.gz
+
+#gene plugins
+gene-data-plugin-places: [mrtarget/plugins/gene]
+gene-data-plugin-names: [HGNC,Orthologs,Ensembl,Uniprot,ChEMBL,MousePhenotypes,Hallmarks,CancerBiomarkers,ChemicalProbes,Tractability]
+
+#hgnc gene plugin
+#TODO move this to GCP https://github.com/opentargets/platform/issues/326
+hgnc-complete-set: http://ftp.ebi.ac.uk/pub/databases/genenames/new/json/hgnc_complete_set.json
```

Comment:
Will this be downloaded and added to the https://storage.googleapis.com/open-targets-data-releases/_releaseTag_/input/annotation-files folder?

Reply:
Yes, see issue opentargets/platform#326